stk/ops/eltwise_ops.py

```diff
@@ -0,0 +1,31 @@
+from stk.matrix import Matrix
+
+def mul(a, b):
+    """Performs element-wise multiplication of matrices a and b.
+
+    It is the user's responsibility to make sure that a and b
+    follow the same matrix topology. This function assumes it is safe
+    to use the topoplogy of a.
+
+    Args:
+        a:
+           sparse stk.matrix.Matrix.
+        b:
+           sparse stk.matrix.Matrix with a's matrix topology.
```

Comment:
The public type here is actually `stk.Matrix` - we expose that API in the `__init__.py` file.

Reply:
nice, fixed!